air-script/tests/aux_trace/aux_trace.masm

```diff
@@ -0,0 +1,42 @@
+proc.compute_evaluate_transitions
+    # constraint 0 for main
+    padw mem_loadw.4294965000 drop drop padw mem_loadw.4294965001 movdn.3 movdn.3 drop drop padw mem_loadw.4294965000 movdn.3 movdn.3 drop drop padw mem_loadw.4294965001 movdn.3 movdn.3 drop drop ext2mul padw mem_loadw.4294965002 movdn.3 movdn.3 drop drop ext2mul ext2add ext2sub
+    # constraint 1 for main
+    padw mem_loadw.4294965001 drop drop padw mem_loadw.4294965002 movdn.3 movdn.3 drop drop padw mem_loadw.4294965000 drop drop ext2add ext2sub
+    # constraint 2 for main
+    padw mem_loadw.4294965002 movdn.3 movdn.3 drop drop padw mem_loadw.4294965000 movdn.3 movdn.3 drop drop padw mem_loadw.4294965001 movdn.3 movdn.3 drop drop ext2add ext2sub
+    # constraint 0 for aux
+    padw mem_loadw.4294965072 drop drop padw mem_loadw.4294965072 movdn.3 movdn.3 drop drop padw mem_loadw.4294965000 movdn.3 movdn.3 drop drop padw mem_loadw.4294965000 movdn.3 movdn.3 drop drop ext2add padw mem_loadw.4294965001 movdn.3 movdn.3 drop drop ext2add padw mem_loadw.4294965000 drop drop ext2add ext2mul ext2sub
+    # constraint 1 for aux
+    padw mem_loadw.4294965073 movdn.3 movdn.3 drop drop padw mem_loadw.4294965073 drop drop padw mem_loadw.4294965002 movdn.3 movdn.3 drop drop padw mem_loadw.4294965000 movdn.3 movdn.3 drop drop ext2add ext2mul ext2sub
+end # END PROC compute_evaluate_transitions
+
+proc.evaluate_transitions
+    exec.compute_evaluate_transitions
+    # Compute the numerator of the constraint polynomial
+    push.0 push.0
+    # Save the accumulator
+    movdn.2 movdn.2
+    # Accumulate the 4-th constraint into the numerator
+    padw mem_loadw.4294966018 movdn.3 movdn.3 drop drop ext2mul ext2add # accumulate the contraint into the numerator
+    # Save the accumulator
+    movdn.2 movdn.2
+    # Accumulate the 3-th constraint into the numerator
+    padw mem_loadw.4294966017 drop drop ext2mul ext2add # accumulate the contraint into the numerator
+    # Save the accumulator
+    movdn.2 movdn.2
+    # Accumulate the 2-th constraint into the numerator
```

Comment:
minor nit: we could avoid the strange English by just saying "Accumulate constraint {i} into the numerator" like in `compute_evaluate_transitions`

Reply:
changed.